docs/user/service-observability.md

```
@@ -0,0 +1,62 @@
+  -p 16686:16686 \  # Jaeger UI
+  -p 4318:4318 \    # OTLP HTTP receiver (for OpenTelemetry traces)
+  jaegertracing/all-in-one:latest
+
+```
+
+> [!NOTE]
+> You can use the default Flightctl tracing configuration with `insecure: true` to send traces to this Jaeger instance over HTTP.
+
+Once the container is running, open your browser and navigate to `http://localhost:16686`
+Once the container is running, open your browser and navigate to <http://localhost:16686>
+Once the container is running, open your browser and navigate to http://localhost:16686
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)